manifests/setup.pp

```diff
@@ -119,7 +119,7 @@
       command => '/bin/postgresql-setup initdb',
       unless  => '/bin/test -d /var/lib/pgsql/data',
     }
-  }->
+  }
```

Comment:
Why do you remove the depencency?

Reply:
because when the variable 'postgresql_embedded' == false, instead of not apply the exec, the puppet execution fails.

Reply:
Then, shouldn't this be changed for a require on exec 'postgres-init' to 'spacewalk-setup'?

Reply:
This is the code:

if $postgresql_embedded == true {
    exec {'postgres-init':
      cwd     => '/root',
      command => '/bin/postgresql-setup initdb',
      unless  => '/bin/test -d /var/lib/pgsql/data',
    }
  }

  exec {'spacewalk-setup':
    cwd         => '/root',
    command     => '/usr/bin/spacewalk.sh',
    refreshonly => true,
    logoutput   => on_failure,
    require     => File['/etc/sysconfig/spacewalk.answer','/usr/bin/spacewalk.sh'],
  }


If you put a dependency inside 'spacewalk-setup' of 'postgres-init' it will fail when variable "$postgresql_embedded" == false because it will not find "exec {'postgres-init':"